### DISCUSSION TITLE
ETA for Dart SDK ?

### DISCUSSION THREAD

**Discussion by tonyhart7** [pocketbase/pocketbase#68] [Q&A] [answered]
I want to know schedule release package for flutter

**Comment by ganigeorgiev** [comment, marked as answer]
I haven't started working on the Dart SDK yet, but I plan to pick it up somewhere within the next couple of weeks. I'm planning support for all Dart platforms (web, mobile and desktop).

**Reply by ganigeorgiev** [reply]
@tonyhart7 Just to let you know that I've just published the PocketBase Dart SDK - https://pub.dev/packages/pocketbase.

The repo of the sdk could be found here - https://github.com/pocketbase/dart-sdk.

**Reply by tonyhart7** [reply]
cool, cant wait to contribute on dark sdk
